renderer/FastBufferDump.cpp

```diff
@@ -25,40 +25,20 @@ FastBufferDump::FastBufferDump(uint32_t num, uint32_t z, buffer_handle_t b, int
 }
 
 void FastBufferDump::prepareFolder() {
-    struct stat st;
-    bool exist = false;
-
-    mEnableDump = false;
-    int ret = stat(kDumpFolder, &st);
-    if (ret < 0) {
-        if (errno != ENOENT) {
-            ALOGE("Failed to access %s:%s", kDumpFolder, strerror(errno));
-            return;
-        }
-    } else {
-        if (!(st.st_mode & S_IFDIR)) {  // not a folder
-            ALOGD("%s exits but it is not a folder, try to recreate it as folder", kDumpFolder);
-            if (unlink(kDumpFolder) < 0) {
-                ALOGE("Can't remove %s:%s", kDumpFolder, strerror(errno));
-                return;
-            }
-        } else {
-            exist = true;
-        }
+    int ret = unlink(kDumpFolder);
```

Comment:
this unlink can be removed. Previous it is used to remove non-folder path. Assume it is always a folder since it is created by this function.